sql-plugin/src/main/scala/org/apache/spark/sql/rapids/RapidsShuffleInternalManagerBase.scala

```diff
@@ -444,7 +444,7 @@ trait VisibleShuffleManager {
  * @param conf
  * @param isDriver
  */
-abstract class ProxyRapidsShuffleInternalManagerBase(
+protected abstract class ProxyRapidsShuffleInternalManagerBase(
```

Comment:
Same question?

Reply:
Removed it. I had misunderstood the bug.